model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -404,17 +406,70 @@ def _get_next_perf_analyzer_config(self, model_variant_name: str,
         perf_analyzer_config.update_config_from_profile_config(
             model_variant_name, self._config)
 
-        model_batch_size = dimension_values.get("max_batch_size", 1)
-        instance_count = dimension_values.get("instance_count", 1)
-
-        concurrency = 2 * model_batch_size * instance_count
+        concurrency = self._calculate_concurrency(dimension_values)
 
         perf_config_params = {'batch-size': 1, 'concurrency-range': concurrency}
         perf_analyzer_config.update_config(perf_config_params)
 
         perf_analyzer_config.update_config(model.perf_analyzer_flags())
         return perf_analyzer_config
 
+    def _calculate_model_batch_size(
+            self, dimension_values: Dict[str, Union[int, float]]) -> int:
+        batch_size = int(dimension_values.get("max_batch_size", 1))
+
+        min_batch_size_set = self._config.get_config(
```

Comment:
set in this case is also confusing. Implying it is part of a set or contains a set. I would either do `is_min_batch_size_set` or `is_min_batch_size_specified`

Reply:
I've changed this (and the others) to `min_batch_size_is_set_by_config`